dipy/workflows/viz.py

```diff
@@ -46,9 +47,14 @@ def run(self, input_files, cluster=False, cluster_thr=15.,
             small animal brains you may need to use something smaller such
             as 2.0. The distance is in mm. For this parameter to be active
             ``cluster`` should be enabled.
-        random_colors : bool, optional
-            Given multiple tractograms have been included then each tractogram
-            will be shown with different color.
+        random_colors : variable str, optional
```

Comment:
Sometimes you request a string and sometimes a list of string.
This is confusing, can you clarify ?

Reply:
Updated. It only needs a string.